tools/flang1/utils/n2rst/CMakeLists.txt

```diff
@@ -16,8 +16,6 @@ target_include_directories(fen2rst
 
 # Convert nroff documentation to Sphinx format.
 
-file(MAKE_DIRECTORY ${FLANG1_DOC_BIN_DIR})
-
```

Comment:
Don't we need to create the directory before writing files to it?

Reply:
`tools/flang1/flang1exe/CMakeLists.txt` also creates the directory, but only when `FLANG_INCLUDE_DOCS`  is true. That may result in situation where the `n2rst` build needs the directory but cannot find it.

Reply:
This directory is created by two other `CMakeLists.txt`s already, but I guess you're right - it doesn't hurt if we keep this command in here, as it generates no errors or warnings and will keep this script working if other `CMakeLists.txt`s alter.

Reply:
Thanks.